samples/ADC/Program.cs

```diff
@@ -1,9 +1,8 @@
 ﻿//
-// Copyright (c) 2018 The nanoFramework project contributors
+// Copyright (c) 2021 The nanoFramework project contributors
```

Comment:
The copyright year it's not meant to be brought "up to date" when the file is touched. Please leave it with the original year.